app/controllers/shipit/merge_status_controller.rb

```diff
@@ -60,9 +60,11 @@ def stack
       @stack ||= if params[:stack_id]
         Stack.from_param!(params[:stack_id])
       else
-        scope = Stack.order(merge_queue_enabled: :desc, id: :asc).where(
-          repo_owner: referrer_parser.repo_owner,
-          repo_name: referrer_parser.repo_name,
+        scope = Stack.order(merge_queue_enabled: :desc, id: :asc).joins(:repository).where(
```

Comment:
Same thing here, `includes(:repository)` would be preferable.

I even wonder if it should be made a default scope of Stack.

Reply:
Resolved by: Resolved in: https://github.com/Shopify/shipit-engine/pull/963/commits/459f3885e65ecd7396d5fabcb86c11cf4033f6fa